docs/changelog/107121.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 107121
+summary: Allow writes to ILM shrink action target index
```

Comment:
What do you think about this? Since this is going to be listed in the release change log it might benefit from a bit more context.
```suggestion
summary: Add a flag to re-enable writes on the final index after an ILM shrink action.
```